wagtail_streamfield_migration_toolkit/utils.py

```diff
@@ -10,17 +10,21 @@ def __init__(self, *args, instance=None, revision=None, **kwargs):
         super().__init__(*args, **kwargs)
 
     def __str__(self):
+        message = ""
+        message += super().__str__()
+        if self.args:
+            message += "\n"
         if self.instance is not None:
-            return "Invalid block def in {} object ({})".format(
+            message += "Invalid block def in {} object ({})".format(
                 self.instance.__class__.__name__, self.instance.id
             )
         elif self.revision is not None:
-            return "Invalid block def in {} object ({}) created at {}".format(
+            message += "Invalid block def in {} object ({}) created at {}".format(
                 self.revision.__class__.__name__,
```

Comment:
This will always be `Revision` - should we not say what model it's a revision of instead?